src/alpha-lab/gaze_contingency_assistive.md

```diff
@@ -0,0 +1,72 @@
+
+## Limitations and current prospects 
+
+Gaze-contingent assistive technologies have become much easier to use recently thanks to advancements in the field of eye tracking. Traditional assistive systems require frequent calibration, which can be problematic in practice, as highlighted in the story of [Gary Godfrey](https://pupil-labs.com/blog/community/cycling-for-als/). Modern calibration-free approaches like [Neon](https://pupil-labs.com/products/neon/) overcome this issue and provide a more robust and user-friendly input modality for producing gaze data.
+
+We have prepared a guide to aid you in creating your very own gaze-contingent assistive applications using [Neon](https://pupil-labs.com/products/neon/).
+
+## How to use a head-mounted eye tracker for screen-based interaction
+
+**Mapping gaze to screen**
+
+Neon is a wearable eye tracker that provides gaze data in scene camera coordinates, i.e., relative to its forward-facing 
+camera. We therefore need to transform gaze from *scene-camera* to *screen-based* coordinates in real time, such that 
+the user can interact with the screen. Broadly speaking, we need to locate the screen, send gaze data from Neon to the 
+computer, and map gaze into the coordinate system of the screen.
+
+To locate the screen, we use [AprilTags](https://april.eecs.umich.edu/software/apriltag) to identify the image of the 
+screen as it appears in Neon’s scene camera. Gaze data is transferred to the computer via Neon's 
+[Real-time API](/neon/real-time-api/introduction/). We then transform gaze from *scene camera* to *screen-based* 
+coordinates using a [homography](https://en.m.wikipedia.org/wiki/Homography_(computer_vision)) approach like the [Marker Mapper](/enrichments/marker-mapper/) 
+enrichment we offer in Pupil Cloud as a post-hoc solution. The heavy lifting of all this is handled by 
+our [Real-time Screen Gaze](https://github.com/pupil-labs/realtime-screen-gaze/) package (written for this guide).
+
+**Gaze-controlling a mouse**
+
+The second challenge is using screen-mapped gaze to control an input device, e.g., a mouse. To demonstrate how to do this, we 
+wrote the [Gaze-controlled Cursor Demo](https://github.com/pupil-labs/gaze-controlled-cursor-demo). This demo leverages the 
+[Real-time Screen Gaze](https://github.com/pupil-labs/realtime-screen-gaze/) package to obtain gaze in screen-based coordinates, and then uses 
+that to control a mouse in a custom browser window, as shown in the video above. A simple dwell-time filter implemented 
```

Comment:
do we need to define dwell time?

Reply:
I don't want to make this longer - so up to the authors to decide if definition is needed.

Reply:
I think it's pretty self-explanatory. Let's leave it.